css-technical-plan.md

```diff
@@ -0,0 +1,131 @@
+  - ⚜ review and update CODE_OF_CONDUCT.md (group task / PM task?)
+- 👨‍💻 add monorepo related files
+  - add top level index file scss in /css
+  - add top level package.json and init monorepo
+- 👨‍💻 implment changesets [changesets](https://github.com/atlassian/changesets)
+- 👨‍💻 implement tokens (each direct sub list item is a dev task)
+  - color tokens
+    - convert tokens to hsla (pending dropping IE theming confirmation)
+    - raw colors in palette.scss
+    - themed colors in color-themes.scss
+  - 👨‍💻 spacing tokens
+  - 👨‍💻 typography, font-size tokens
+  - 👨‍💻 depth/shadow tokens
+  - 👨‍💻 write tokens readme
+- 👨‍💻 implement code formatting and linting
+  - requirements:
+    - code format on save
+    - fast via cli
+    - pre-commit hook w/husky
+  - proposed combination: prettier and stylelint combo, favoring airbnb stylelint presets
+- 👨‍💻 implment mixins folder
+  - first move media query mixins
+  - wait until we need a mixin before moving something else
+  - catalog shared and shareable mixins from docs-ui
+- implement Core folder (each sub item is a dev task)
+  - 👨‍💻 resets (normalize and minireset + custom?)
+  - 👨‍💻 create css custom properties for theme colors
+  - 👨‍💻 implement font stack
+    - no font face rules
+    - just native font family (from github / bulma)
+    - this will be overridden downstream
+  - 👨‍💻 implement focus styling (focus-visible) 💭
+    - escape hatch, which can toggle :focus-visible, rather than the polyfill (.focus-visible)
+  - 👨‍💻 implement a css build for prototyping 💭
+    - should include a font face?
+- devops setup
+  - 👨‍💻 github actions on pull_request open
+    - lint (to ensure prepush cannot be avoided and files are in sync)
+  - 👨‍💻 release workflow 💭
+    - tag and publish on pr to specific branch?
+- implement Atomics folder (each sub item is a dev task)
+  - 👨‍💻 spacing atomic classes
+  - 👨‍💻 depth atomic classes
+  - 👨‍💻 typography atomic classes
+  - 👨‍💻 color atomic classes
+  - 💭 standardize on has
+- create components folder
+  - create components folder readme and index file
+  - Will we be implementing any components as part of this feature 💭
+    - 👨‍💻 columns
+    - 👨‍💻 nav-bar?
+- 👨‍💻 plugins (top level folder)
+  - this folder contains postcss plugins
+  - how are plugins structured? 💭
+  - each plugin is a package
+  - all live in packages folder
+  - implment post css plugin
+  - move theme fallback plugin 👨‍💻
+
+## Folders in /css
+
+The majority of this work will be contained within the /css folder.
+
+### Src
+
+Each folder has an index file and a README that explains the nature of the folder. See below.
+
+Proposed initial folder structure.
+
+```txt
+css
+└───src
+    ├───tokens
+    ├───mixins
+    ├───core
+    ├───atomics
+    └───components
+```
+
+#### Tokens
+
+- No output
+- Shared variables
+- Colors
+- Typography
+
+#### Mixins
+
+- No output
+- Depends on tokens
+- Contains shared mixins and functions
+
+#### Core
+
+- output producing
```

Comment:
Is it worth noting "output producing" for "Atomics" and "Components", or is it implied/intuitive?

Reply:
I think it's worth noting, I'll do that